pybuildkite/constants.py

```diff
@@ -0,0 +1,9 @@
+import enum
+
+
+AGENT_V3_BASE_URL = "https://agent.buildkite.com/v3"
+
+
+class APIVersion(enum.Enum):
+    V2 = "v2"
+    V3 = "v3"
```

Comment:
I like the idea of having a version enum but it looks like we aren't using it for the base url.